R/pin-read-write.R

```diff
@@ -144,21 +144,21 @@ object_write <- function(x, path, type = "rds") {
 }
 
 write_rds <- function(x, path) {
-  if (!is_testing()) {
-    saveRDS(x, path, version = 2)
-  } else {
-    # compression algorithm changed in 4.1
-    saveRDS(x, path, version = 2, compress = FALSE)
+  saveRDS(x, path, version = 2)
+  invisible(path)
+}
 
-    old <- readBin(path, "raw", fs::file_size(path))
+write_rds_test <- function(x, path) {
```

Comment:
This is only used in one test, but the function is called directly so I can't seem to mock it. If I do this:

```r
local_mocked_bindings(write_rds = write_rds_test)
```
it does not mock the results when I call `write_rds()`. Is that right?